src/components/EventListCard/EventListCard.test.tsx

```diff
@@ -14,11 +17,33 @@ const MOCKS = [
     },
     result: {
       data: {
-        organizations: [
-          {
-            _id: '1',
-          },
-        ],
+        removeEvent: {
+          _id: '1',
+        },
+      },
+    },
+  },
+  {
+    request: {
```

Comment:
why is this dummy data

Reply:
This dummy data is to mock graphql queries used in EventListCard.tsx file.